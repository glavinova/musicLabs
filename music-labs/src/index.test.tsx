import ReactDOM from "react-dom";

jest.mock("react-dom/client", () => ({
  createRoot: jest.fn().mockImplementation(() => ({
    render: jest.fn(),
  })),
}));

describe("index.tsx unit test", () =>{
  it("renders with App and root div", () => {
    const root = document.createElement("div");
    root.id = "root";
    document.body.appendChild(root);
    require("./index.tsx");
    //expect(ReactDOM.render).toHaveBeenCalled();
  });
})

