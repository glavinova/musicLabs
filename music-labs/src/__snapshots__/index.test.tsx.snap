// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index.tsx unit test Should render app without crashing 1`] = `"{\\"_reactInternalInstance\\":\\"censored\\"}"`;
